packages/ui/react-ui/src/WalletSVG.tsx

```diff
@@ -0,0 +1,51 @@
+        <path d="M61.4231 46.6172H59.577C58.8123 46.6172 58.1924 47.2371 58.1924 48.0018C58.1924 48.7665 58.8123 49.3863 59.577 49.3863H61.4231C62.1878 49.3863 62.8077 48.7664 62.8077 48.0018C62.8077 47.2371 62.1878 46.6172 61.4231 46.6172Z" fill="url(#paint3_linear_880_5115)"/>
+        </g>
+        <defs>
+        <linearGradient id="paint0_linear_880_5115" x1="3.41664" y1="98.0933" x2="103.05" y2="8.42498" gradientUnits="userSpaceOnUse">
+        <stop stopColor="#9945FF"/>
+        <stop offset="0.14" stopColor="#8A53F4"/>
+        <stop offset="0.42" stopColor="#6377D6"/>
+        <stop offset="0.79" stopColor="#24B0A7"/>
+        <stop offset="0.99" stopColor="#00D18C"/>
+        <stop offset="1" stopColor="#00D18C"/>
+        </linearGradient>
+        <linearGradient id="paint1_linear_880_5115" x1="3.41664" y1="98.0933" x2="103.05" y2="8.42498" gradientUnits="userSpaceOnUse">
+        <stop stopColor="#9945FF"/>
+        <stop offset="0.14" stopColor="#8A53F4"/>
+        <stop offset="0.42" stopColor="#6377D6"/>
+        <stop offset="0.79" stopColor="#24B0A7"/>
+        <stop offset="0.99" stopColor="#00D18C"/>
+        <stop offset="1" stopColor="#00D18C"/>
+        </linearGradient>
+        <linearGradient id="paint2_linear_880_5115" x1="25.9583" y1="68.7101" x2="67.2337" y2="23.7879" gradientUnits="userSpaceOnUse">
+        <stop stopColor="#9945FF"/>
+        <stop offset="0.14" stopColor="#8A53F4"/>
+        <stop offset="0.42" stopColor="#6377D6"/>
+        <stop offset="0.79" stopColor="#24B0A7"/>
+        <stop offset="0.99" stopColor="#00D18C"/>
+        <stop offset="1" stopColor="#00D18C"/>
+        </linearGradient>
+        <linearGradient id="paint3_linear_880_5115" x1="58.3326" y1="49.4467" x2="61.0002" y2="45.4453" gradientUnits="userSpaceOnUse">
+        <stop stopColor="#9945FF"/>
+        <stop offset="0.14" stopColor="#8A53F4"/>
+        <stop offset="0.42" stopColor="#6377D6"/>
+        <stop offset="0.79" stopColor="#24B0A7"/>
+        <stop offset="0.99" stopColor="#00D18C"/>
+        <stop offset="1" stopColor="#00D18C"/>
+        </linearGradient>
+        <clipPath id="clip0_880_5115">
+        <rect width="48" height="48" fill="white" transform="translate(24.5 24)"/>
+        </clipPath>
+        </defs>
+        </svg>
+    )
+}
```

Comment:
I think we can remove this SVG, see https://github.com/solana-labs/wallet-adapter/pull/214#issuecomment-999081376 for the latest designs from Josh's Figma file. If we can avoid using colors and graphics, this will fit in better with more dapp designs and be easier to customize.

Reply:
Yes, this SVG is used in the "no wallets detected" screen, which is consistent with the design. We have tried to keep the colors relatively neutral. 

![image](https://user-images.githubusercontent.com/16849485/147856509-cedf9812-99c2-496a-adcc-a746a92dd39f.png)